rs-serverless-resizing/serverless.yml

```
@@ -0,0 +1,31 @@
+service: rs-serverless-image-resizing
+provider:
+  name: aws
+  runtime: rust
+  region: ap-southeast-1
+  iamRoleStatements:
+    -  Effect: "Allow"
+       Action:
+         - "s3:GetObject"
+       Resource: "arn:aws:s3:::image-upload-bucket-staging/*"
+    -  Effect: "Allow"
+       Action:
+         - "s3:PutObject"
+       Resource: "arn:aws:s3:::images-to-be-rated-staging/*"
+plugins:
+  # this adds informs serverless to use
+  # the serverless-rust plugin
+  - serverless-rust
+# creates one artifact for each function
+package:
+  individually: true
+functions:
+  test:
```

```suggestion
  rs-img-resizing-func
```

If I understand correctly, this is the name of the function. 
We should probably go through the `serverless` documentation to better understand and get a good grasp of what exactly is going on in this file.